src/sentry/api/endpoints/monitor_checkins.py

```diff
@@ -137,9 +146,22 @@ def post(
         result = serializer.validated_data
 
         with transaction.atomic():
+            environment = Environment.get_or_create(
+                project=project, name=result.get("environment", "production")
```

Comment:
@mitsuhiko wanted to confirm, you mentioned environments without a name would default to `production` in most SDKs correct.

In sentry we default environments without a name to the `''` empty string. We're proactively going to use `production` as the default here, since this won't be sent via SDKs yet.

Does this behavior sound right to you?

Reply:
Ah also this logic is slightly wrong, need to ensure `result.get()` doesn't return `None` as field is nullable